docs/CustomCommands.md

```diff
@@ -40,20 +44,20 @@ it('should use my custom command', () => {
 __Note:__ if you register a custom command to the browser scope the command won't be accessible for elements. Likewise, if you register a command to the element scope, it won't be accessible at the browser scope:
 
 ```js
-browser.addCommand("myCustomBrowserCommand", function () { return 1});
+browser.addCommand("myCustomBrowserCommand", function () { return 1 });
 const elem = $('body')
 console.log(typeof browser.myCustomBrowserCommand); // outputs "function"
 console.log(typeof elem.myCustomBrowserCommand()); // outputs "undefined"
 
-browser.addCommand("myCustomElementCommand", function () { return 1 }, true);
+browser.addCommand("myCustomElementCommand2", function () { return 1 }, true);
```

Comment:
why `myCustomElementCommand2`? There was not `myCustomElementCommand`

Reply:
elem2 confused me, fixing